### PR TITLE
#5 [ADD] User Database Schema

### PR DESCRIPTION
**Pull request by MKB-1**
Created the full schema for a user account.

### CODE REVIEW

**Review by loren589 — APPROVED** [review]
Looks good to me.

File: models/User.js [hunk 1]
```diff
@@ -1,5 +1,33 @@
...
-const UserSchema = new mongoose.Schema({});
+const UserSchema = new mongoose.Schema({
+    avatar: {
+        type: String
+    },
+    date: {
+        type: Date,
+        default: Date.now
+    },
+    email: {
```
Comment (gaticosalas) [inline models/User.js:11]: add
unique: true
to email field

**Review by gaticosalas — CHANGES_REQUESTED** [review]
Everything looks good just add that unique requirement for the email

File: models/User.js (hunk 1 above)
Comment (MKB-1) [reply models/User.js:11]: You're right, good catch!

**Review by gaticosalas — APPROVED** [review]
Looks great

**Pull request merged**